packages/components/src/date-time/date.js

```diff
@@ -35,10 +36,24 @@ class DatePicker extends Component {
 		onChange( newDate.set( momentTime ).format( TIMEZONELESS_FORMAT ) );
 	}
 
+	/**
+	 * Create a Moment object from a date string. With no currentDate supplied, default to a Moment
+	 * object representing now. If a null value is passed, return a null value.
+	 *
+	 * @param {string|null} [currentDate] - Date representing the currently selected date or null to signify no selection.
```

Comment:
While I think this is technically valid JSDoc, it doesn't really follow the conventions from the rest of the project, specifically:

- Using `?` to signify a nullable type
- Omitting the `[` `]` brackets around the variable name
- Omitting the `-` delimiter between the variable name and description.

```suggestion
	 * @param {?string} currentDate Date representing the currently selected
	 *                              date or null to signify no selection.
```